src/mono/wasm/testassets/WasmBasicTestApp/App/wwwroot/main.js

```diff
@@ -127,6 +127,19 @@ switch (testCase) {
         };
         dotnet.withConfig({ maxParallelDownloads: maxParallelDownloads });
         break;
+    case "AllocateLargeHeapThenInterop":
```

Comment:
In the test triggering this scenario, `AllocateLargeHeapThenRepeatedlyInterop` we have
`string extraArgs = BuildTestBase.IsUsingWorkloads ? "-p:EmccMaximumHeapSize=4294901760" : "-p:EmccMaximumHeapSize=4294901760";`
I know I added this line but it does not make sense.

Reply:
please take over :)